src/room.py

```diff
@@ -1,2 +1,18 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+class Room:
+    def __init__(self, name, description, items=[]):
+        self.name = name
+        self.description = description
+        self.items = items
+        self.n_to =  None
```

Comment:
self.n_to =  None and its siblings are redundant. the room already inherited the attributes already.

Reply:
Technically, you are right, however i choose to init them to none, because at the time of initial instantiation of the room class the n_to. e_to... fields do not exist. Its probably not important